src/main/java/com/epam/ta/reportportal/core/launch/impl/StartLaunchHandlerAsyncImpl.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.ta.reportportal.core.launch.impl;
+
+import com.epam.ta.reportportal.commons.ReportPortalUser;
+import com.epam.ta.reportportal.core.launch.StartLaunchHandler;
+import com.epam.ta.reportportal.ws.model.launch.StartLaunchRQ;
+import com.epam.ta.reportportal.ws.model.launch.StartLaunchRS;
+import com.epam.ta.reportportal.ws.rabbit.MessageHeaders;
+import org.springframework.amqp.core.AmqpTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static com.epam.ta.reportportal.core.configs.RabbitMqConfiguration.QUEUE_LAUNCH_START;
+
+/**
+ * @author Konstantin Antipin
+ */
+@Service
+@Qualifier("startLaunchHandlerAsync")
+public class StartLaunchHandlerAsyncImpl implements StartLaunchHandler {
+
+    @Autowired
+    @Qualifier(value = "rabbitTemplate")
+    AmqpTemplate amqpTemplate;
+
+    @Override
+    /**
+     * @return {@link List < TestItem >}
+     */
```

Comment:
What does it mean if method return type - StartLaunchRS?

Reply:
fixed